active/tab/tabcontrol.md

```diff
@@ -0,0 +1,401 @@
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+> See [proposal document](https://github.com/Microsoft/microsoft-ui-xaml/issues/304) for more background
+
+The Tab control is a way to display a set of tabs and their respective content. Tab controls are useful for displaying several pages (or documents) of content while giving a user the capability to rearrange, open, or close new tabs. 
+
+![Tabs in Edge](https://user-images.githubusercontent.com/25991996/52679758-f6d6ad80-2eea-11e9-9955-fd111c26d982.png)
+
+Tab-like UI comes in two distinct styles that affect not only the visualization of the control, but also the user experience:
+* Static tabs
+    * Generally display a set number of pages
+    * User can change between pages, but cannot open new pages, close pages, or rearrange pages
+* Document tabs
+    * Display a variable number of pages
+    * User can add, remove, and rearrange pages
+    * User can "tear off" tabs into their own window or "recombine" tabs from one group into another group
+
+The Xaml platform already provides several controls to represent "static-style tabs" including [Pivot](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/pivot) and top [NavigationView](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#display-modes). However, these controls do not well support "document-style" tabs due to a variety of limitations, including:
+* Significant effort up to (and including) retemplating to build simple tabs
+* No built-in support for closing tabs
+* No built-in support for drag/drop into new windows
+* No built-in support for moving a tab from a window and combining it with another window
+* Limited keyboard and accessibility support
+
+This spec therefore covers "document-style tabs" specifically as this platform gap isn't easily solved with an app-side workaround. 
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+The Tab control is a collection of tabs that each represents a new page or document in your app. Tab controls are useful when your app has several pages of content and the user expects to be able to add, close, and rearrange the pages.
+
+**Is this the right control?**
+
+Use a TabView to help the user manage multiple app pages or documents within the same window. 
+
+Do not use a TabView to display a static set of tabs that the user cannot rearrange, open, or close. Use a Pivot or top NavigationView instead. 
```

Comment:
The name "TabView" doesn't imply this restriction. Have you discussed some more specific name?

Reply:
This restriction is purely a guidance recommendation and not a restriction in the API. We briefly discussed other options like "DocumentTabView", "BrowserTabView", "DynamicTabView", etc. 

However, these more specific names have a number of drawbacks, including:
- Discoverability: not starting with the word "Tab" will make it more difficult to find this control
- Coupling with data type: The words "Document" and "Browser" suggest specific data items. For example, DocumentTabView.ItemsSource would probably only take a collection of <Document>. TabView doesn't have this restriction.

Perhaps the wording in the guidance could be updated, as there are legitimate "static tab" scenarios that _could_ be satisified by TabView instead of Pivot/Top NavView. 

To help demonstrate, I built a very basic Ribbon using TabView. (Ribbon has its own set of requirements and scenarios, but I built this to demonstrate that the current TabView is robust enough to be worthy of the name TabView 😉)
https://github.com/stmoy/TabViewSamples/tree/master/RibbonUsingTabView

![image](https://user-images.githubusercontent.com/25991996/61147256-c8a4cc00-a490-11e9-89c7-aad0897f65c2.png)